### PR TITLE
Update troubleshooting steps with Windows steps

### PR DESCRIPTION
**Pull request by hestonhoffman**
Updates Logs troubleshooting instructions for Linux and adds instructions for Windows.

### Motivation

See [DOCS-4576](https://datadoghq.atlassian.net/browse/DOCS-4576)

### Preview
Watch for a git comment below with a link to the preview.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-4576]: https://datadoghq.atlassian.net/browse/DOCS-4576?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/logs/guide/log-collection-troubleshooting-guide.md [hunk 1]
```diff
@@ -58,41 +58,79 @@ The Datadog Agent only collects logs that have been written after it has started
...
+   In this example, the `application` directory is not executable by the Agent.
+
+1. Run this PowerShell script to give read and execute privileges to `ddagentuser`:
+   ```powershell
+   $acl = Get-Acl <path\to\logs\folder>
+   $AccessRule = New-Object System.Security.AccessControl.FileSystemAccessRule("ddagentuser","ReadAndExecute","Allow")
+   $acl.SetAccessRule($AccessRule)
+   $acl | Set-Acl <path\to\logs\folder>
+   ```
+
```
Comment (clarkb7) [inline content/en/logs/guide/log-collection-troubleshooting-guide.md:125]: could maybe show the `get-acl` output again with the permissions added.

**Review by clarkb7 — APPROVED** [review]

**Review by clarkb7 — APPROVED** [review]
LGTM for windows

File: content/en/logs/guide/log-collection-troubleshooting-guide.md [hunk 2]
```diff
@@ -58,41 +58,79 @@ The Datadog Agent only collects logs that have been written after it has started
...
-```
-
-In this example, the `application` directory is not executable by the Agent, therefore the Agent cannot list its files. Furthermore, the Agent does not have read permissions on the `error.log` file.
-Add the missing permissions via the [chmod command][6].
-
-{{< img src="logs/agent-log-permission-ok.png" alt="Permission OK"  style="width:70%;">}}
-
-**Note**: When adding the appropriate read permissions, make sure that these permissions are correctly set in your log rotation configuration. Otherwise, on the next log rotate, the Datadog Agent may lose its read permissions.
-Set permissions as `644` in the log rotation configuration to make sure the Agent has read access to the files.
+The Datadog Agent does not run as root (and running as root is not recommended, as a general best-practice). When you configure your Agent to tail log files for custom logs or for integrations, you need to take special care to ensure the Agent user has the correct access to the log files.
```
Comment (cswatt) [inline content/en/logs/guide/log-collection-troubleshooting-guide.md:61]:
```suggestion
The Datadog Agent does not run as root (and running as root is not recommended, as a general best practice). When you configure your Agent to tail log files for custom logs or for integrations, you need to take special care to ensure the Agent user has the correct access to the log files.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
just a tiny nit

**Review by clarkb7 — APPROVED** [review]

**Review by cswatt — APPROVED** [review]

**Pull request merged**